sdks/corda/src/main/kotlin/com/weaver/corda/sdk/AssetManager.kt

```diff
@@ -104,6 +107,78 @@ class AssetManager {
             }
         }
 
+        @JvmStatic
+        @JvmOverloads fun createFungibleAssetPledge(
+            proxy: CordaRPCOps,
+            localNetworkID: String,
+            remoteNetworkID: String,
+            tokenType: String,
+            numUnits: Long,
+            recipientCert: String,
+            expiryTimeSecs: Long,
+            getAssetStateAndRefFlow: String,
+            deleteAssetStateCommand: CommandData,
+            issuer: Party,
+            observers: List<Party> = listOf<Party>()
+        ): Either<Error, String> {
+            return try {
+                AssetManager.logger.debug("Sending fungible asset pledge request to Corda as part of asset-transfer.\n")
+                val contractId = runCatching {
+                    val assetAgreement = createFungibleAssetExchangeAgreement(tokenType, numUnits, "", "")
```

Comment:
I thought we decided to create new class to pass these values from SDK to Workflows, instead of using HTLC protos?

Reply:
Added now.